autoscheduler/scraper/tests/models_tests.py

```diff
@@ -36,8 +39,103 @@ class MeetingTests(unittest.TestCase):
     def test_generate_meeting_id_returns_correct_id(self):
         """ Tests that section.generate_meeting_id correctly generates id """
 
-        # Arrange
+        # Act
         meeting_id = generate_meeting_id("123456", "0")
 
         # Assert
         self.assertEqual(meeting_id, "1234560")
+
+class GradesTests(django.test.TestCase):
+    """ Tests for Grade model + GradeManager """
+
+    def test_instructor_performances_calculates_from_all_terms(self):
+        """ Tests that instructor performance uses all of the
+            sections from different terms
+        """
+
+        # Arrange
+        instructor = Instructor(id="First Last", email_address="last@tamu.edu")
+        instructor.save()
+        subject = "CSCE"
+        course_num = 121
+
+        sections = [
+            Section(id=10, subject=subject, course_num=course_num, instructor=instructor,
+                    term_code=201931, section_num=500, min_credits=3,
+                    current_enrollment=0, max_enrollment=10),
+            Section(id=11, subject=subject, course_num=course_num, instructor=instructor,
+                    term_code=201831, section_num=500, min_credits=3,
+                    current_enrollment=0, max_enrollment=10),
+        ]
+
+        Section.objects.bulk_create(sections)
+
+        grades = [
+            Grades(section=sections[0], gpa=4.0, A=1, B=0, C=0, D=0, F=0, I=0, S=0, U=0,
+                   Q=0, X=0),
+            Grades(section=sections[1], gpa=3.0, B=1, A=0, C=0, D=0, F=0, I=0, S=0, U=0,
+                   Q=0, X=0),
+        ]
+
+        # Save all at once

```

Comment:
Really minor but this comment should be above the first `bulk_create` instead of this one

Reply:
Honestly I'm just gonna remove this comment. I think `bulk_create` is pretty straightforward, plus it's a really easy search to see what it does in the case it's not.